src/reformat.c

```diff
@@ -1081,18 +1081,22 @@ avifResult avifImageYUVToRGB(const avifImage * image, avifRGBImage * rgb)
         return AVIF_RESULT_REFORMAT_FAILED;
     }
 
+    avifAlphaMultiplyMode alphaMultiplyMode = state.toRGBAlphaMode;
     avifBool convertedWithLibYUV = AVIF_FALSE;
-    avifResult libyuvResult = avifImageYUVToRGBLibYUV(image, rgb);
-    if (libyuvResult == AVIF_RESULT_OK) {
-        convertedWithLibYUV = AVIF_TRUE;
-    } else {
-        if (libyuvResult != AVIF_RESULT_NOT_IMPLEMENTED) {
-            return libyuvResult;
+    avifResult libyuvResult = AVIF_RESULT_NOT_IMPLEMENTED;
```

Comment:
Please delete line 1086:

```
    avifResult libyuvResult = AVIF_RESULT_NOT_IMPLEMENTED;
````

and declare libyuvResult directly at line 1088:

```
        avifResult libyuvResult = avifImageYUVToRGBLibYUV(image, rgb);
````

The blank line at 1089 could be deleted.

Reply:
Done.
Tip: You can also push commits to the branch in my fork directly, if you prefer.

Reply:
Thank you. I will modify your pull requests directly to make this kind of small change in the future. (I recently worked with someone who preferred to make all the changes I suggested, big or small.)

By the way, I think it is good to write a pull request now to use only the I420ToRGB24() function.